pkg/account/component.go

```diff
@@ -31,6 +32,8 @@ const (
 	emailSubjectUpdatedEmail     = "notifEmailChangeSubject"
 	emailTemplateUpdatedProfile  = "notif-profile-change.ftl"
 	emailSubjectUpdatedProfile   = "notifProfileChangeSubject"
+
+	frontendType = "sse"
```

Comment:
Should this constant not be defined at the endpoint level ? 
Minor thing : I also challenge the name, sse refers to self-service that is a webapp. This api can be used by other frontend. Is it really a type ?

Reply:
If one day, a client wants to use this method to implement his own interface, I might not need our self-service... So we will be able to define his configuration using the "sse" value.
If the client wants to have our self-service AND its own self-service with different attributes, then we will have to find a solution... but I don't think this will be done in the future. (and if he wants to implements its own self-service, I'm not sure he will have to have a dynamic configuration)

Reply:
I think you did not really answer my first point : I do not really see the point of defining this constant in component.go and endpoint.go. I think we will always define this constant in endpoint (example KYC). So I think we can only define it in endpoint.go.


Maybe I should rephrase the second point : I challenged the name, because we are in a api called account and now we are binding it with the frontendType. In my point of view, the bridge is not aware of the front-end and is do not need to be aware of  which frontend is calling him. So why putting a parameter called sse ? I think this is misleading and the parameter should have the name of the api.

Reply:
I can move declaration in endpoint... it was created where it was used, in component (it was first used only there). Now endpoint also use it.

I don't really care about the name (you can make suggestions... frontendName?)... I think the most important is the value...
The attribute management is clearly designed to be adapted for a specific context. I just wanted to help integrators to be able to configure realms the most simply as possible: management is too long and you can easily swap to letters. How should we write: self service, self-service, self_service ? To avoid doubt, I used sse.

Reply:
if we want to put : sse, bo ... as values frontendType is ok for me.

I would suggest to have apiName with the name of the api. I think it is not good to over simplify our system for integrator. It is only my personal point of view. If you do not agree and @harture is fine with it, it is okay to keep as it is now.